package.json

```diff
@@ -22,6 +22,7 @@
 		"IS_GUTENBERG_PLUGIN": true
 	},
 	"dependencies": {
+		"@ariakit/react": "0.4.5",
```

Comment:
Note that the dep is declared in the components package, not in the root package.json.

Reply:
Fixed, thanks for the heads up.